doc/src/pair_eam.rst

```diff
@@ -140,6 +140,20 @@ The OpenKIM Project at
 provides EAM potentials that can be used directly in LAMMPS with the
 :doc:`kim command <kim_commands>` interface.
 
+.. warning::
+
+   The EAM potential files tabulate the embedding energy as a function
+   of the local electron density :math:`\rho`.  When atoms get too
+   close, this electron density may exceed the range for which the
+   embedding energy was tabulated for.  For simplicity and to avoid
+   errors during equilibration of randomized geometries, LAMMPS will
+   assume a linearly increasing embedding energy for electron densities
+   beyond the maximum tabulated value.  This usually means that the EAM
```

Comment:
Suggest rewording last three sentences to "LAMMPS will print a single warning if the maximum tabulated value is exceeded, which usually indicates that the EAM model is not a good model for the kind of system under investigation.  It may be harmless at the beginning of an equilibration but is a big concern for accuracy if it happens during production runs."